docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -0,0 +1,225 @@
+
+## Create, Retrieve, and Deploy a Custom Field
+Let’s add a custom field to an object in our org and pull its metadata into our Code Builder project.
+
+First let’s add a custom field  – 
+
+1. From **Setup**, go to **Object Manager** | **Account**.
+2. Click **Fields & Relationships**.
+3. Click **New**.
+4. For data type, select **Date**, then click **Next**.
+5. Fill out the following:
+     * Field Label: Created On
+     * Field Name: createdon
+     * Description: Date of Account Creation
+6. Click **Next** until you save the field.
+
+Now let’s retrieve metadata for this new field –
+
+1. Return to Code Builder and open Org Browser.
+2. Scroll down to **Custom Objects** and navigate to **Account**.
+3. Click the retrieve icon next to the Account component to run **SFDX: Retrieve Source from Org**.
+4. From the Activity Bar, click the Explorer and navigate to `force-app/main/default/object/Account`
+Lo and behold, in the `fields` folder, a file named `createdon_c.field-meta.xml` contains metadata for your new custom field! 
+
+The metadata is here for your reference:
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+   <fullName>createdon__c</fullName>
+   <description>Date of Account Creation</description>
+   <externalId>false</externalId>
+   <label>Created On</label>
+   <required>false</required>
+   <trackFeedHistory>false</trackFeedHistory>
+   <type>Date</type>
+</CustomField>
+```
+We’ll now make a simple edit to this field and deploy our changes back to our org with a single click.
+
+1. Edit `createdon_c.field-meta.xml` and change the `<required>` tag value to `true` to indicate that this custom field is required. 
+2. Right click the `objects/Account` folder and click **SFDX: Deploy Source to Org**. 
+3. After the command has successfully run, go back to your org and check details of the **Created On** custom field and confirm that it’s now a required field. 
+
+## Create and Deploy a New Lightning Web Component
+For another exercise in deploying, let’s learn how to quickly create a simple Lightning Web Component in our Code Builder project. We’ll then deploy this component to our org using a single command. Here we go:
+
+Let’s create a Lightning Web Component – 
+
+1. Press Ctrl+Shift+P (⇧⌘P) run **SFDX: Create Lightning Web Component** to create a New Lightning Web Component. 
+2. Give the component a name, say, `newCBComponent`.
+3. Press **Enter** to accept the default file location (`force-app/main/default/lwc`).
+4. Press **Enter**.
+
+Three new files are created in the `force-app/main/default/lwc/newCBComponent` folder.
+
+Update the files –
+
+1. In the HTML file, `newCBComponent.html`, copy and paste the following code:
+
+```
+<template>
+    <p>Hello, {greeting}!</p>
+    <lightning-input label="Name" value={greeting} onchange={handleChange}></lightning-input>
+</template>
+```
+2. In the HTML file, `newCBComponent.js`, copy, and paste the following code:
```

Comment:
Numbering in preview in off. Take a look. I wonder if using all "1."s for numbered steps would fix it.